### PR TITLE
Update NAP Version

### PR DESCRIPTION
**Pull request by rafwegv**
### Proposed changes
Update NAP Version in Dockerfiles and add bot-defense to CRDs

### Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto master
- [x] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by pleshakov** [comment]
Hi @rafwegv . Could you possibly update CRDs used in the helm chart as well? 

I think it makes sense to rename AppProtect CRDs in the `helm/crds` folder as well, so that they have the same names as in `deployments/common/crds-v1beta1`

File: tests/suite/test_app_protect_integration.py [hunk 1]
```diff
@@ -334,8 +334,12 @@ def test_ap_sec_logs_on(
         assert f'outcome="REJECTED"' in log_contents
 
         print("----------------------- Send valid request ----------------------")
+        headers = {
```
Comment (pleshakov) [inline tests/suite/test_app_protect_integration.py:337]: is this change required for this PR? 🤔

**Review by pleshakov — APPROVED** [review]

File: tests/suite/test_app_protect_integration.py (hunk 1 above)
Comment (rafwegv) [reply tests/suite/test_app_protect_integration.py:337]: The new version of app protect introduced bot-signatures. Any typical non browser user-agent strings in the header (curl python etc.)  will be classified as "non-browser request" , so this particular test started failing. In order to make the request valid the user agent was set to one of a firefox browser.

Comment (pleshakov) [reply tests/suite/test_app_protect_integration.py:337]: @rafwegv thanks for clarifying!

cc @vepatel so he's aware

**Review by Dean-Coakley — APPROVED** [review]

File: tests/suite/test_app_protect_integration.py (hunk 1 above)
Comment (vepatel) [reply tests/suite/test_app_protect_integration.py:337]: @pleshakov already had discussion with @rafwegv re the same.

**Review by lucacome — APPROVED** [review]
